test/actions/pulp/repository/sync_test.rb

```diff
@@ -11,10 +11,6 @@ class VCRTestBase < TestBase
     include VCR::TestCase
     let(:repo) { katello_repositories(:fedora_17_x86_64) }
 
-    def run_action(action_class, *args)
```

Comment:
Does removing this method cause failures in other tests that may depend on this?